src/main/webapp/app/shared/exercise-filter/exercise-filter-modal.component.scss

```diff
@@ -0,0 +1,19 @@
+::ng-deep ngb-typeahead-window.dropdown-menu {
+    max-height: 25rem !important;
+    overflow-y: auto !important;
+}
+
+.no-left-margin-padding:first-child {
+    margin-left: 0 !important;
+    padding-left: 0 !important;
+}
+
+// otherwise the dropdown changes the color on hover if no further options can be selected
+.form-control:disabled,
+.form-control[disabled] {
+    pointer-events: none;
+}
+
+label {
+    cursor: pointer;
+}
```

Comment:
Made the cursor a pointer

Reply:
I specifically added a global `pointer` class since it's needed quite often. You could use it here. :)